CardForm/src/main/java/com/braintreepayments/cardform/view/InitialValueCheckBox.java

```diff
@@ -0,0 +1,63 @@
+package com.braintreepayments.cardform.view;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.util.AttributeSet;
+
+import androidx.appcompat.widget.AppCompatCheckBox;
+
+/**
+ * Sets an initial CheckBox checked state that is overwritten when restoring this view.
+ */
+public class InitialValueCheckBox extends AppCompatCheckBox {
+    private static final String EXTRA_SUPER_STATE = "com.braintreepayments.cardform.view.DefaultValueCheckBox.EXTRA_SUPER_STATE";
```

Comment:
The value should be: `com.braintreepayments.cardform.view.InitialValueCheckBox.EXTRA_SUPER_STATE`